webapp/pages/post/_id/_slug/index.spec.js

```diff
@@ -143,5 +144,48 @@ describe('PostSlug', () => {
         })
       })
     })
+
+    describe('tags shown in tag cloud', () => {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-03T14:45:57Z" title="Wednesday, June 3rd 2020, 4:45:57 pm +02:00">Jun 3, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -143,5 +144,48 @@ describe('PostSlug', () => {
         })
       })
     })
+
+    describe('tags shown in tag cloud', () => {
```


Nicely done 😍
Thanks for testing! 👍🏼